src/navigation/apiNavigation.js

```diff
@@ -15,6 +15,7 @@ const nav = [
   { title: 'Loyalty Programs' },
   { title: 'Media Uploads' },
   { title: 'Merchants' },
+  { title: 'Notifications' },
```

Comment:
@Ray2126 Change: This should be called "Webhooks" to match standard terminology that developers expect

Reply:
They are called Notifications in our system though. If we expose APIs to retrieve notifications, I would expect them to be called "List Notifications", "Get Notification" etc. rather than "List Webhooks".

They are called [Events](https://docs.stripe.com/api/events) in Stripe so I'm not sure we need to conform to the Webhook terminology.

Reply:
But the API to subscribe might be called "Create Webhook"? Or "Create Notification Subscription"?

I like keeping the language here consistent with our system to avoid confusion. Thoughts?

Reply:
@Ray2126 Events are the payload, not the feature. The webhook terminology is important - I don't know of a single tool that doesn't use it and we actively look for it ourselves when evaluating new vendors 👍  

Stripe's webhook guide is here -> https://docs.stripe.com/webhooks

Reply:
The language I had in mind was:
- Developers create a **webhook endpoint** in their system
- A webhook endpoint can be registered with Centrapay by **subscribing**  it to one or more **events**
- Once subscribed, a webhook endpoint will re./ive **notifications** (requests) when those events occur
- A notification contains an the event payload and other metadata (delivery status etc)

So "Listing notifications" would list all deliveries to your webhook endpoint

Reply:
As discussed, we will separate Events from Notifications in API reference section.